app/actions/actions.js

```diff
@@ -22,6 +24,7 @@ export const indicatePageWasLoaded = (page) => {
  * @param history Object used to navigate from one page to another (accessible in props usually).
  */
 export const changePage = (fromPage, toPage, history) => {
+	history.push("/") //reposition at root - hacky, i know
```

Comment:
TODO: see if my baseurl fix improves this behavior so i can get rid of this hack.